### PR TITLE
Potato Cells Properly Described (FFF17)

### LINKED ISSUE #16419: Potato batteries are borked
-potato batteries power is supposed to increase with potency
-a potency 110 potato had the default 300 charge, should have been much higher, potato contained no chemicals
-potency 64 potato with very high levels of nutriment also resulted in a 300 charge battery

I read somewhere in the code prior a long time ago that potato battery cells are supposed to scale with potency (albeit poorly), this didn't happen.

### PR DESCRIPTION
**Pull request by Kurfursten**
closes #16419

🆑 
* bugfix: Potato power cells will now properly describe how much maximum charge is within.

### CODE REVIEW

**Comment by SonixApache** [comment]
I was actually on the verge of doing this myself, very nice.

File: code/modules/power/cell.dm [hunk 1]
```diff
@@ -58,10 +54,12 @@
...
 /obj/item/weapon/cell/examine(mob/user)
 	..()
-	if(crit_fail)
-		to_chat(user, "<span class='warning'>This power cell seems to be faulty.</span>")
+	if(!starch_cell)
+		to_chat(user, "The manufacturer's label states this cell has a power rating of [maxcharge], and that you should not swallow it.")
+		to_chat(user, "<span class='info'>The charge meter reads [crit_fail ? "NAN" : round(percent())]%.</span>")
 	else
-		to_chat(user, "<span class='info'>The charge meter reads [round(src.percent() )]%.</span>")
+		to_chat(user, "Based on the starchiness, it's probably has a maximum potential of [maxcharge].")
```
Comment (Intigracy) [inline code/modules/power/cell.dm]: `it's probably has`

File: code/modules/power/cell.dm [hunk 2]
```diff
@@ -58,10 +54,12 @@
 
 /obj/item/weapon/cell/examine(mob/user)
 	..()
-	if(crit_fail)
```
Comment (jknpj) [inline code/modules/power/cell.dm:61]: what happened with crit_fail?

Comment (Kurfursten) [reply code/modules/power/cell.dm:61]: it's now part of a ternary statement

**Pull request merged**